src/lib/schematics/dashboard/index.spec.ts

```diff
@@ -1,48 +1,42 @@
 import {SchematicTestRunner} from '@angular-devkit/schematics/testing';
-import {join} from 'path';
-import {createTestApp} from '../utils/testing';
+import {createTestApp, collectionPath} from '../utils/testing';
 import {getFileContent} from '@schematics/angular/utility/test';
-
-const collectionPath = join(__dirname, '../collection.json');
+import {Schema} from './schema';
 
 describe('material-dashboard-schematic', () => {
   let runner: SchematicTestRunner;
-  const options = {
+  const options: Schema = {
     name: 'foo',
-    path: 'app',
-    sourceDir: 'src',
-    inlineStyle: false,
-    inlineTemplate: false,
+    project: 'material',
     changeDetection: 'Default',
     styleext: 'css',
     spec: true,
-    module: undefined,
     export: false,
-    prefix: undefined,
-    viewEncapsulation: undefined,
   };
 
   beforeEach(() => {
     runner = new SchematicTestRunner('schematics', collectionPath);
   });
 
-  it('should create dashboard files and add them to module', () => {
+  // Temporarily disabled because @angular-devkit/schematics does not properly resolve
+  // the files that should be copied to the project.
```

Comment:
Is there a link to an issue?

Reply:
Not that I know off. And I'm not sure whether this is actually something the CLI project needs to resolve just in favor of Bazel testing. It might just also depend on bazelbuild/rules_typescript#154